org.eclipse.jdt.core.tests.builder/src/org/eclipse/jdt/core/tests/builder/Java50Tests.java

```diff
@@ -112,9 +113,9 @@ public void testHierarchyCycleInstanceof() throws JavaModelException {
 	}
 
 	// https://bugs.eclipse.org/bugs/show_bug.cgi?id=231293
-	public void testMissingRequiredBinaries() throws JavaModelException {
+	public void _2551_testMissingRequiredBinaries() throws JavaModelException {
 
```

Comment:
Not immediately obvious how this test differs from the others in this suite that caused it to be disabled ? Could you add a comment explaining ?

Reply:
> Not immediately obvious how this test differs from the others in this suite that caused it to be disabled ? Could you add a comment explaining ?

Perhaps this comes under the catch all task `Create ticket to analyze bugs disabled via "2551" prefix` ? (I think you mean tests not bugs?)

Reply:
No, in this case I didn't manage to fix the test and that means I didn't understand why it failed. In other cases I've noted on this PR TODO list why something is broken.

Reply:
> think you mean tests not bugs

Sure, corrected

Reply:
I've started test to see where it fails - it was supposed to find some buildpath issue because of incomplete classpath but now it doesn't see the problem. No idea why. I would not add comment in the code, since it would not add any additional info. Whoever checks the test fail later would have to run the test anyway and then will see real failure not some unqualified comment about it.